test/cmd/images.sh

```diff
@@ -134,6 +134,16 @@ os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.t
 os::cmd::expect_success 'oc tag mysql:5.5 newrepo:latest --alias'
 os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}'" 'ImageStreamTag'
 
+# test scheduled and insecure tagging
+os::cmd::expect_success 'oc tag --source=docker mysql:5.7 newrepo:latest --scheduled'
+os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).importPolicy.scheduled}}'" 'true'
+os::cmd::expect_success_and_text "oc describe is/newrepo" '\* tag is scheduled'
+os::cmd::expect_success 'oc tag --source=docker mysql:5.7 newrepo:latest --insecure'
+os::cmd::expect_success_and_text "oc describe is/newrepo" '\! tag is insecure'
+os::cmd::expect_success_and_not_text "oc describe is/newrepo" '\* tag is scheduled'
+oc get -o yaml is/newrepo
```

Comment:
It was a debug line, it can be removed.

On Mon, Mar 14, 2016 at 1:06 PM, Steve Kuznetsov <notifications@github.com>
wrote:

> In test/cmd/images.sh
> <https://github.com/openshift/origin/pull/6685#discussion_r56036282>:
>
> > @@ -134,6 +134,16 @@ os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.t
> >  os::cmd::expect_success 'oc tag mysql:5.5 newrepo:latest --alias'
> >  os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}'" 'ImageStreamTag'
> >
> > +# test scheduled and insecure tagging
> > +os::cmd::expect_success 'oc tag --source=docker mysql:5.7 newrepo:latest --scheduled'
> > +os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).importPolicy.scheduled}}'" 'true'
> > +os::cmd::expect_success_and_text "oc describe is/newrepo" '\* tag is scheduled'
> > +os::cmd::expect_success 'oc tag --source=docker mysql:5.7 newrepo:latest --insecure'
> > +os::cmd::expect_success_and_text "oc describe is/newrepo" '\! tag is insecure'
> > +os::cmd::expect_success_and_not_text "oc describe is/newrepo" '\* tag is scheduled'
> > +oc get -o yaml is/newrepo
>
> @smarterclayton <https://github.com/smarterclayton> was the intent of
> this line to check for existence?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6685/files#r56036282>.
>